mbed.h

```diff
@@ -18,6 +18,14 @@
 
 #define MBED_LIBRARY_VERSION 123
 
+#define MBED_MAJOR_VERSION 5
+#define MBED_MINOR_VERSION 2
+#define MBED_MICRO_VERSION 0
```

Comment:
So I infer you want "patch" _instead of_ "micro" (naming which I had lifted from glib).  A few extra words would have saved us all some time!